channels/offline-channel-js/package.json

```diff
@@ -28,9 +28,9 @@
     "dependencies": {
         "@microsoft/dynamicproto-js": "^2.0.3",
         "@microsoft/applicationinsights-shims": "3.0.1",
-        "@microsoft/applicationinsights-core-js": "3.0.7",
-        "@microsoft/applicationinsights-common": "3.0.7",
-        "@nevware21/ts-utils": ">= 0.10.4 < 2.x",
+        "@microsoft/applicationinsights-core-js": "3.0.8",
+        "@microsoft/applicationinsights-common": "3.0.8",
+        "@nevware21/ts-utils": ">= 0.10.5 < 2.x",
         "@nevware21/ts-async": ">= 0.3.0 < 2.x"
```

Comment:
"@microsoft/api-extractor": "^7.18.19", need to be updated in this file also to 7.40.0